docs/operator/troubleshooting.md

```diff
@@ -0,0 +1 @@
+TBD
```

Comment:
It might be better to leave this section out altogether (unless you're planning on adding something here imminently)

Reply:
Good point. I'm removing it for now and will add it back once we get some feedback from end users.